src/scalib_ext/scalib/src/sasca/fg_parser.rs

```diff
@@ -71,9 +72,11 @@ fn parser() -> impl Parser<char, Vec<Statement>, Error = Simple<char>> {
     let kw = |s| text::keyword::<_, _, Simple<char>>(s).delimited_by(pad, space);
     let var = ident.map(|s| Var(s));
     let not_var = || op('!').ignore_then(var);
+    let negate_var = || op('-').ignore_then(var);
     let nvar = || {
         var.map(|v| NVar::new(v, false))
             .or(not_var().map(|v| NVar::new(v, true)))
+            .or(negate_var().map(|v| NVar::new(v, true)))
```

Comment:
This changes makes `-VAR` and `!VAR` exactly identical, which is not what we want: `a + !b` should either be rejected (that's the current behavior), or be the sum of a and the bit-negation of b.

If we want (and support!) the possibility of applying mod nc negation in many expression, we need to extend NVar to have multiple status (not negated, bit-negated, sign-negated).